content/docs/reference-react-dom.md

```diff
@@ -1,114 +1,113 @@
----
+﻿---
 id: react-dom
 title: ReactDOM
 layout: docs
 category: Reference
 permalink: docs/react-dom.html
 ---
 
-?? ????? React ?? ???? ?????? ?`<script>?` ?????? ??? ??????? ??? ??????? ?????? ?????? ??? ?????? ????? React. ??? ??????? ES6 ?? npm ??????? ????? ?`import React from 'react'`?. ?? ??????? ES5 ?? npm ??????? ????? ?`var React = require('react')?`.
+إن حمّلت React عن طريق العنصر ‎`<script>‎ `فستكون هذه الواجهة ذات المستوى الأعلى متوفرة عبر الكائن العام React. وإن استخدمت ES6 مع npm فتستطيع كتابة ‎`import React from 'react'`‎. إن استخدمت ES5 مع npm فتستطيع كتابة ‎`var React = require('react')`‎.
 
-## ???? ???? {#overview}
+## لمحة عامة {#overview}
 
-????????? ?????? `react-dom` ?????? ????? ?? DOM ????? ?????? ????????? ?? ??????? ?????? ?? ?????? ??????? ???? ?????? ?? ????? React ?? ???? ???. ????? ???? ????? ???? ???????? ??? ??????? ??? ??????.
+تُزوِّدنا الحزمة `react-dom` بتوابع خاصّة بـ DOM والتي يُمكِن استخدامها في المستوى الأعلى من تطبيقك وكوسيلة هروب للخروج من نموذج React إن أردت ذلك. ينبغي ألّا تحتاج معظم مكوّناتك إلى استخدام هذه الوحدة.
 
-- [`render()`](#render)
-- [`hydrate()`](#hydrate)
-- [`unmountComponentAtNode()`](#unmountcomponentatnode)
-- [`findDOMNode()`](#finddomnode)
-- [`createPortal()`](#createportal)
+- [`()render`](#render)
+- [`()hydrate`](#hydrate)
+- [`()unmountComponentAtNode`](#unmountcomponentatnode)
+- [`()findDOMNode`](#finddomnode)
+- [`()createPortal`](#createportal)
 
-### ??? ??????? {#browser-support}
+### دعم المتصفح {#browser-support}
 
-???? React ???? ????????? ???????? ??? ?? ??? Internet Explorer 9 ??? ???? ??? ?????? ??? ????? ??? ????????? ??????? ??? Internet Explorer 9 ? Internet Explorer 10.
+تدعم React جميع المتصفحات الشائعة، بما في ذلك Internet Explorer 9 فما فوق، رغم الحاجة إلى توفير دعم للمتصفحات القديمة مثل Internet Explorer 9 و Internet Explorer 10.
```

Comment:
Original line "React supports all popular browsers, including Internet Explorer 9 and above, although [some polyfills are required](/docs/javascript-environment-requirements.html) for older browsers such as IE 9 and IE 10."